code/style.css

```diff
@@ -0,0 +1,314 @@
+/* ---------------------- GENERAL STYLING ---------------------- */ 
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@500&display=swap');
+
+* {
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+    -webkit-font-smoothing: antialiased;
+    text-rendering: optimizeLegibility;
+}
+
+html {
+    font-family: 'Roboto Mono', monospace;
+    background: #000;
+    color: #fff;
+}
+
+body {
+    padding: 0;
+    width: 70vw;
+    margin: auto;
+    text-shadow: 1px 1px #004D16;
+}
+
+h1 {
+    font-size: 40px;
+}
+
+h2 {
+    margin: 0 auto 30px auto;
```

Comment:
We can use a shorthand for that : `margin: 0 auto 30px` will do the same.
Basically, CSS analyses it this way : 
1. Ok, `0` will go for `top` and `bottom` values for now, until they will tell me later on during this line that I should apply different amount for `bottom`.
2. Later, I have `auto` for `left` and `right` until they will tell me later during this line that I should apply different amount for `left`.
3. Next, I have `30px`, so that one clearly goes for `bottom` as I expected.
4. And then there is nothing, no 4th values. Hmm, so I have `0 auto 30px`. I will apply `0` for `top`, `auto` for both `left` and `right` and at the end `30px` for `bottom`